src/backend/db.js

```diff
@@ -1,11 +1,17 @@
 const Pool = require('pg').Pool;
+require("dotenv").config();
+
+const host = process.env.PSQL_HOST;
+const password = process.env.PSQL_PASSWORD;
+const user = process.env.PSQL_USER;
+const database = process.env.PSQL_DATABASE;
 
 const pool = new Pool({
-    user: "postgres",
-    password: "123",
-    host: "localhost",
+    user: user,
```

Comment:
You can use this JS shorthand for `user`, `password`, `host` and `database`

```suggestion
    user,
    password,
    host,
```